examples/column_transformer_mixed_types.py

```diff
@@ -0,0 +1,101 @@
+"""
+===================================
+Column Transformer with Mixed Types
+===================================
+
+This example demonstrates how to use
+:class:`sklearn.compose.ColumnTransformer` on a dataset consisting of
+hereogeneous data types. A subset of numeric and categorical features
+are selected and :class:`sklearn.compose.ColumnTransformer` is used to
+apply two non-sequential preprocessing pipelines, one for each data
```

Comment:
how about saying that different pipelines are used for different subsets of the features and the results are concatenated? I find "non-sequential" a bit odd here.